### DISCUSSION TITLE
Add a somewhat brighter mode

### DISCUSSION THREAD

**Discussion by MattievandenHeuvel** [ownbee/doorstop-edit#6] [Ideas]
Because of the dark mode the text in the document, level and header fields are barely readable unless you select them. It would be nice to be able to switch to a mode that isn't that dark, or at least all text are readable when not selected

**Comment by ownbee** [comment]
Sorry for a very late response but I don't see that problem in my environment (Linux), can you post a screenshot of the problem? What OS are you using?
